src/couch/test/couch_bt_engine_upgrade_tests.erl

```diff
@@ -16,14 +16,18 @@
 -include_lib("couch/include/couch_db.hrl").
 
 
-setup() ->
+setup(_) ->
```

Comment:
Why did this change?

Reply:
because i used `foreachx` to perform test from 6 to 8, and 7 to 8 respectively now. So just follow the case in https://github.com/apache/couchdb/blob/103a0624f309ea0d796176a55eb5faea68f26047/src/couch_replicator/test/couch_replicator_filtered_tests.erl#L49

Reply:
Gotchya. Sounds good. EUnit is hard.